VultisigApp/VultisigApp/Views/Send/SendCryptoVerifyView.swift

```diff
@@ -210,11 +210,11 @@ struct SendCryptoVerifyView: View {
     }
     
     private func getAmount() -> String {
-        tx.amount + " " + tx.coin.ticker
+        tx.amount.formatCurrencyWithSeparators(settingsViewModel.selectedCurrency) + " " + tx.coin.ticker
     }
     
     private func getFiatAmount() -> String {
-        tx.amountInFiat.formatToFiat()
+        tx.amountInFiat.formatToFiat().formatCurrencyWithSeparators(settingsViewModel.selectedCurrency)
```

Comment:
![image](https://github.com/user-attachments/assets/37a04908-bc08-4d10-8cc6-4de95c7e36d9)
Refer to here , the FiatAmount is empty, I am using USD as my currency

Reply:
Fixed the formatter's fallback value.
![Simulator Screenshot - iPhone 15 Pro - 2024-11-06 at 18 08 17](https://github.com/user-attachments/assets/5d9dfc8e-0a79-49a9-a10b-d7f7c596184f)